packages/server/src/manifest-generator.ts

```diff
@@ -5,59 +5,86 @@ import { throwOnErrorEvent } from '@effection/events';
 import * as fs from 'fs';
 import * as globby from 'globby';
 import * as path from 'path';
+import { EslintValidator } from './validators/eslint-validator';
+import { Validator, OrchestratorState, BundlerState } from './orchestrator/state';
+import { Atom, Slice } from '@bigtest/atom';
 
 const { writeFile, mkdir } = fs.promises;
 
 interface ManifestGeneratorOptions {
   delegate: Mailbox;
   files: string[]; 
   destinationPath: string;
+  atom: Atom<OrchestratorState>;
 };
 
-function* writeManifest(options: ManifestGeneratorOptions) {
-  let files = yield globby(options.files);
+function* writeManifest({ bundlerSlice, destinationPath, ...options }: ManifestGeneratorOptions & { validator: Validator; bundlerSlice: Slice<BundlerState, OrchestratorState> }) {
```

Comment:
Can we just add these to the ManifestGeneratorOptions and slice off the bundler state inside this method?

Reply:
`validator` and `bundlerSlice` are only used for `writeManifest`.  The `ManifestGeneratorOptions` is used in orchestrator so I created a new type.